### PR TITLE
Fixes refresh, fixes post-create redirect, adds cancellation to new references/acknowledgments

### PR DESCRIPTION
**Pull request by superbuggy**

### CODE REVIEW

File: src/composables/useFlawModel.ts [hunk 1]
```diff
@@ -44,18 +39,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), emit: FlawEmitt
   const osimLink = computed(() => getFlawOsimLink(flaw.value.uuid));
 
   async function createFlaw() {
-    postFlaw(flaw.value)
+    isSaving.value = true;
```
Comment (C-Valen) [inline src/composables/useFlawModel.ts:42]: Isn't `isSaving` being handled twice for update/create actions, as it is still on `FlawForm` `onSubmit`?

File: src/composables/useFlawModel.ts [hunk 2]
```diff
@@ -44,18 +39,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), emit: FlawEmitt
...
       .then((response: any) => {
         router.push({
-          name: 'flaw-detail',
-          params: { id: response?.uuid },
+          name: 'flaw-details',
+          params: { id: response?.cve_id || response?.uuid },
         });
       })
       .catch(createCatchHandler('Error creating Flaw'));
+    isSaving.value = false;
```
Comment (C-Valen) [inline src/composables/useFlawModel.ts:52]: Not sure about this, but I'm thinking on the proper way to ensure `isSaving = false` is set in every scenario.
```suggestion
      .catch(createCatchHandler('Error creating Flaw'))
      .finally(() => { isSaving.value = false; });
```

File: src/composables/useFlawModel.ts [hunk 3]
```diff
@@ -44,18 +39,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), emit: FlawEmitt
...
   }
 
   async function updateFlaw() {
+    isSaving.value = true;
```
Comment (C-Valen) [inline src/composables/useFlawModel.ts:56]: Same case a above:
> Isn't isSaving being handled twice for update/create actions, as it is still on FlawForm onSubmit?

**Review by C-Valen — COMMENTED** [review]
Generally looks good, just small details that may be worth considering.

File: src/components/IssueFieldAcknowledgments.vue [hunk 4]
```diff
@@ -45,10 +46,18 @@ function handleDelete(uuid: string, closeModal: () => void) {
...
-            from
-            <input v-model="items[itemIndex].affiliation" type="text" />
+          <div class="d-flex justify-content-between">
+            <div>
+              <input v-model="items[itemIndex].name" type="text" />
+              from
+              <input v-model="items[itemIndex].affiliation" type="text" />
+            </div>
+            <div
+              class="text-end  osim-cancel-new-acknowledgment"
```
[the remark this answers is not in the document]
Comment (superbuggy) [reply src/components/IssueFieldAcknowledgments.vue:56]: No :( I have a separate PR for this #125

File: src/composables/useFlawModel.ts (hunk 1 above)
Comment (superbuggy) [reply src/composables/useFlawModel.ts:42]: Yes, thank you!

File: src/composables/useFlawModel.ts (hunk 2 above)
Comment (superbuggy) [reply src/composables/useFlawModel.ts]: Yes, I can ditch the await above.

**Review by C-Valen — APPROVED** [review]
🚀

**Pull request merged**